src/Mollie/WC/Helper/IconFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Mollie\Api\Types\PaymentMethod;
+
+class Mollie_WC_Helper_IconFactory
+{
+
+    /**
+     * @param $data
+     * @return string
+     */
+    public function create($data)
+    {
+        if (!preg_match('/[^A-Za-z0-9]/', $data)) {
+            $methods = $this->getApiMethods();
+            if(!$methods){
+                return $this->fallbackToAssetsFolder($data);
+            }
+
+            foreach ($methods as $method){
+                if($method['id'] == $data){
+                    return $method['image']->svg;
+                }
+            }
+        }
+        //Problems
+        return $data;
```

Comment:
What about this? it will return a string which is not the file path of the payment method icon right?
Also, what the comment means? `Problems`?